### PR TITLE
explicitly call out OS and requirements at top of OS X install doc

### PR DESCRIPTION
**Pull request by arubdesu**
Even though I’m unsure myself, since I noticed there’s no mention of 10.11 support or really any sysReqs, I added it to the docs. (May want to bump the readme with that info as well, didn’t feel like I should be making that call.) 
Pardon the presumption of accuracy 😅

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

File: docs/wiki/installation/install-osx.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## System Requirements
+
+osquery has no package or library dependencies on OS X.
```
Comment (theopolis) [inline docs/wiki/installation/install-osx.md]: This is an interesting statement for the top of the "install" docs. Technically... the Homebrew/custom-build install methods have quite a few dependencies. The package install has none! 

It is very valuable to communicate that the package has no dependencies, but at the same time, most packages don't have dependency lists/requirements.

File: docs/wiki/installation/install-osx.md [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+### Supported OS Versions
+
+Continuous integration currently tests stable release versions of osquery against 10.9 and 10.10 (as listed under the _Build_status_ column on the project [README](https://github.com/facebook/osquery/blob/master/README.md)). There are no reported issues which block expected core functionality on 10.11.  
+Each tagged release of osquery may be installed on all versions of OS X.
```
Comment (theopolis) [inline docs/wiki/installation/install-osx.md]: This is a very important, but previously missed, comment.

**Comment by theopolis** [comment]
Maybe we can swap the order on the install page and introduce the **OS X Package** method first? Then merge in the note about CI on 10.9-11 as another paragraph under that heading and ditch the duplicate line w.r.t. dependencies? :dancer:

**Comment by arubdesu** [comment]
Agreed about the reorder, swapped things around. Not sure about the manual addition of line breaks, I let 'em flow.

**Comment by theopolis** [comment]
Looking good! If you don't mind squashing these three commits into 1, I'll merge ASAP!

**Comment by arubdesu** [comment]
er, I think I've made it worse... should I extract the actual commit and start over? _(Reminds self to sync/merge the changes in master before I make a topic branch next time)_

**Comment by theopolis** [comment]
Haha, yeah, it looks like something messed up. If you have the `git reflog` you can back up before your most recent amend, as you should see the commit hash before/after each action. If you can cherry-pick your updates you can then `git rebase -i HEAD~3` and choose 's' for squash before each of your commits. Or you can backup to master, make your intended changes, and `git push -f` them as a new single commit back to the PR.

**Comment by arubdesu** [comment]
aHA! http://stackoverflow.com/questions/3810348/setting-git-parent-pointer-to-a-different-parent and a `push -f` (hold onto your butts) and it's looking there!

**Comment by theopolis** [comment]
Who da bomb? You da :bomb:!

**Pull request merged**